README.md

```diff
@@ -15,6 +15,11 @@ make bootstrap
 make test
 ```
 
+You can use [tox](https://tox.readthedocs.io) to run tests as well.
+```bash
+tox
+```
```

Comment:
What does tox do to get the required versions of Python?

Reply:
As far as I know, it does nothing, so you have to have required versions of Python already installed.
Alternatively, you can run tests against some specific versions which are installed, e.g. `tox -e py27,py36`